pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -738,6 +738,10 @@ func (adc *attachDetachController) processVolumeAttachments() error {
 						err)
 					continue
 				}
+				klog.V(5).Infof("CSI Migration has been enabled for volume:%q, va.Name:%q, nodeName:%q",
```

Comment:
Can be modified to klog.V(5).InfoS

Reply:
Yeah, I understand it is better to use structure logging. However, the entire file has been using Infof. So I think it would be best to do this as part of a bigger effort in another commit